demos/jans-tarp/src/popup.js

```diff
@@ -96,17 +97,25 @@ const qs = require('qs');
 
   async function trigCodeFlowButton() {
 
-    const redirectUrl = chrome.identity.getRedirectURL()/*chrome.runtime.getURL('redirect.html')*/
+    const redirectUrl = chrome.identity.getRedirectURL()
+    const { secret, hashed } = await generateRandomChallengePair();
+
     await chrome.storage.local.get(["oidcClient"]).then(async (result) => {
```

Comment:
There is no need to add await keyword please remove that.

Reply:
done